docs/adding-to-project.md

```diff
@@ -1,9 +1,8 @@
 # Adding BLT to an existing project
```

Comment:
I am missing a paragraph on what benefits do I have if I add BLT to an existing project.